README.md

```diff
@@ -12,24 +12,51 @@ anyone can use this project to run said accelerators on the cloud and
 develop host code (such as drivers and userspace libraries) targetting
 said accelerators.
 
+# Overview
+
+This project consists of three main parts.
+
+## QEMU Model
+
+This part of the project will develop and ideally upstream a
+hermes-based PCIe device model. This code will, initially, be hosted
+in a public fork of [QEMU][4]. Some early work on such a model results
+in [this RFC][5].
+
+## AWS F1 Implementation
+
+This part of the project implements an eid-hermes bitstream which can
+run on an AWS F1 instance. This code will be contained in this repo
+and will consist mostly of RTL.
+
+## eid-hermes Linux driveer
+
+This part of the project implements a Linux driver for eid-hermes and
+should threfore work for both the QEMU model and the AWS F1 instance
+noted above. This code *may* be hosted in this repo initially and
+ideally will be upstreamed into the [Linux kernel][6].
+
```

Comment:
I'd say there's a fourth part too, which is the userspace library.

Reply:
Added @iomartin